tools/android.py

```diff
@@ -18,19 +18,15 @@ def options(opts):
 
 
 def exists(env):
-    if not "ANDROID_NDK_ROOT" in os.environ:
-        return False
-    return True
+    return "ANDROID_NDK_ROOT" in os.environ or "ANDROID_NDK_ROOT" in ARGUMENTS
 
 
 def generate(env):
-    if not "ANDROID_NDK_ROOT" in os.environ:
+    if "ANDROID_NDK_ROOT" not in env:
```

Comment:
Won't this break support for the OS environment variable?

Reply:
It shouldn't since the option is always parsed, and the option default is `os.environ.get("ANDROID_NDK_ROOT")`:

https://github.com/godotengine/godot-cpp/pull/797/files/0943dfc34a75436e754b6f56c7ed2a061df99585#diff-2a1ce0ab3a735a0ed135b46f3c36535f7c9824ecefc81b91134f2678c4a5d407R13-R17

Reply:
Ah ok, I missed that this was an option already.